### PR TITLE
Cypress integration test for Magento2

### PR DESCRIPTION
**Pull request by saadidrees711**

### CODE REVIEW

File: Test/Integration Test/cypress/integration/examples/payment.js [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+
+    it('CC Payment', function(){
+
+        const ord = new Order()
+        ord.clrcookies()
+        ord.visit()
+        ord.signin()
+        // ord.email_field('saadidrees57@gmail.com')
+        // ord.pass_field('admin@123')
+        // ord.login_btn()
```
Comment (emicha) [inline Test/Integration Test/cypress/integration/examples/payment.js]: Remove commented lines

File: Test/Integration Test/cypress/integration/examples/payment.js [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+
+    it('Klarna Payment', function(){
+
+        const ord = new Order()
+        ord.clrcookies()
+        ord.visit()
+        ord.signin()
+        // ord.email_field('saadidrees57@gmail.com')
+        // ord.pass_field('admin@123')
+        // ord.login_btn()
```
Comment (emicha) [inline Test/Integration Test/cypress/integration/examples/payment.js]: Remove commented lines

File: Test/Integration Test/cypress/integration/examples/payment.js [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+
+    it('Subscription', function(){
+
+        const ord = new Order()
+        ord.clrcookies()
+        ord.visit()
+        ord.signin()
+        // ord.email_field('saadidrees57@gmail.com')
+        // ord.pass_field('admin@123')
+        // ord.login_btn()
```
Comment (emicha) [inline Test/Integration Test/cypress/integration/examples/payment.js]: Remove commented lines

File: Test/Integration Test/Readme.md [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+{
+  "url": "http://sampledomain.com",
+  "email": "demo@mail.com",
+  "pass": "password",
+  "adminURL":"http://sampledomain.com/admin",
+  "adminUsername":"admin_user",
+  "adminPass":"admin_password"
+}
+
+3) Run `npm run cypress_run` to run all the tests
```
Comment (emicha) [inline Test/Integration Test/Readme.md]: Add this to the main README.md instead in the section 
## Integration Tests
Make sure than you use a developer friendly guide, with less test and more commands, so it is very easy to spot what needs to be done.

## Steps

1) Install dependencies
```bash
npm i
```
2) Update `cypress/fixtures/config.json`
3) Run test 
```bash
npm run cypress_run <- Please provide actual path
```

File: Test/Integration Test/cypress/integration/PageObjects/objects.js [hunk 5]
```diff
@@ -98,10 +108,14 @@ class Order
             cy.get('.action-login').click().wait(2000)
             })
 
+<<<<<<< HEAD
+
+=======
```
Comment (AJenbo) [inline Test/Integration Test/cypress/integration/PageObjects/objects.js]: you have a conflict

**Comment by saadidrees711** [comment]
Conflict has been resolved in the latest push. Please check now.

**Review by emicha — APPROVED** [review]

**Pull request merged**